### ISSUE TITLE
Atmos mix tank control console input/output issues

### ISSUE THREAD

**Issue by Polyphynx** [tgstation/tgstation#35004]
Issue reported from  Round ID: 82615 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

This only happens on every map except Box Station. It seems to work just fine on Box.

The Gas Mix Tank Control console is broken all around:
-Switching the input on and off turns both the input and output on and off.
-Pressing the output button doesn't change the state on the console, but still switches both the input and output on and off, again.
-Changing the output pressure doesn't change anything on the console, staying stuck on 0kPa. I don't know if the pressure is even changing at all.
-Reconnecting the console to the mix tank makes the output button do nothing at all.

Basically, there's no way to have the input on with the output off, and vice versa, making actually mixing the gases awkward to do. This is only on the Mix tank control console, the others work fine.

**Comment by Polyphynx** [comment]
This was fixed at some point.
